react/Modal.tsx

```diff
@@ -64,7 +64,14 @@ const Modal: React.FC<Props> = props => {
     }
   }
 
-  const containerClasses = classnames(styles.containerCenter, handles.container, 'bg-base relative flex flex-column')
+  const containerClasses = classnames(
+    styles.containerCenter,
+    handles.container,
+    'bg-base relative flex flex-column',
```

Comment:
```suggestion
    'bg-base relative flex flex-column w-100 mw-100 h-100 br0',
```

Reply:
These classes are part of the `fullScreen` visualization, so I can't apply them here